File: 백준/Silver/1543. 문서 검색/문서 검색.java
import java.io.*;

public class Main {

    private static String inputStr;
    private static String targetStr;
    private static int answer;

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        inputStr = br.readLine();
        targetStr = br.readLine();
        answer = 0;

        int inputStrLen = inputStr.length();
        int targetStrLen = targetStr.length();

        String replacedStr = inputStr.replace(targetStr, "");
        int replacedStrLen = replacedStr.length();

        answer = (inputStrLen - replacedStrLen) / targetStrLen;
        System.out.println(answer);
    }
}

File: 백준/Silver/1543. 문서 검색/문서 검색.kt
fun main() = with(System.`in`.bufferedReader()) {

    val inputStr = readLine();
    val targetStr = readLine();
    val inputStrLen = inputStr.length;
    val targetStrLen = targetStr.length;
    
    val replacedStr = inputStr.replace(targetStr, "");
    val replacedStrLen = replacedStr.length;

    val answer = (inputStrLen - replacedStrLen) / targetStrLen;
    println(answer);
}